src/ethereum_test_tools/eof/v1/eof_classes.py

```diff
@@ -0,0 +1,114 @@
+"""
+EOF v1 specs
+https://github.com/ipsilon/eof/blob/main/spec/eof.md
+"""
+
+from typing import Generic, TypeVar, List
+
+class Byte:
```

Comment:
If you really need these two classes, I think an off-the-shelf solution could be to use the `base_types.FixedSizeBytes` class, you can declare something like:
```python
from ..common.base_types import FixedSizeBytes

class Byte(FixedSizeBytes[1]):  # type: ignore
    """
    Class that helps represent a single byte.
    """

    pass


class Bytes2(FixedSizeBytes[2]):  # type: ignore
    """
    Class that helps represent two bytes.
    """

    pass
```

An advantage of this is that these play nicely with pydantic.